conf/routes

```diff
@@ -9,6 +9,7 @@ GET     /home                                                @controllers.Applic
 GET     /api                                                 @controllers.ApplicationController.developer
 GET     /developer                                           @controllers.ApplicationController.developer
 GET     /terms                                               @controllers.ApplicationController.terms
+GET     /maintenance                                         @controllers.ApplicationController.maintenance
```

Comment:
So including this is useful for testing, but I think that it should be removed for the PR. When we need to use this page, we will make all the other URLs load this page instead. For example, we would just change `@controllers.ApplicationController.terms` above to `@controllers.ApplicationController.maintenance`. This would make it so that when someone goes to sidewalk-newberg-test.cs.washington.edu/terms, they would see the maintenance page at that URL. But we don't need the maintenance page available at any other time.